src/System.Device.Gpio/System/Device/Gpio/PinVector32.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Device.Gpio
+{
+    /// <summary>
+    /// 32-bit vector of pins and values.
+    /// </summary>
+    public struct PinVector32
+    {
+        /// <summary>
+        /// Bit vector of pin numbers from 0 (bit 0) to 31 (bit 31).
+        /// </summary>
+        public uint Pins { get; set; }
```

Comment:
I think this should be called something like PinValueMask (something with Mask in the name I think)

Reply:
Which? The struct itself or the member? Naming for the struct was meant to align with what we current have for `BitVector32`. I tried more complicated wording for the properties but it got a bit cumbersome, imho.